suricata/update/main.py

```diff
@@ -193,14 +198,14 @@ def fetch(self, url):
             tmp_fileobj.close()
         except URLError as err:
             if os.path.exists(tmp_filename):
-                logger.warning(
+                strict_error(
```

Comment:
This will log an `error` now instead of a warning. I'm thinking it does not go with the message. The message indicates a warning. Should I keep it a warning or modify the message a bit?

Reply:
Thats fine. Its probably better as an error than a warning anyways.